Scripts_BeyzanurÖzen/Door.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Door : MonoBehaviour, IUsable
+{
+
+	public GameObject UIText;
+
+	public void Text()
+	{
+		UIText.SetActive(true);
+	}
+	public void Action()
+	{
+		if (Input.GetKey("e"))

```

Comment:
Input checki player'in update loop unda yapilarak IUsabledan Action methodu cagirilmaliydi